Casks/voicevox.rb

```diff
@@ -0,0 +1,28 @@
+cask "voicevox" do
+  version "0.14.7"
+  sha256 "341df00cced9f8b516dda4e79902d4106a34f89dc18264cb1bb6a5721282d75c"
+
+  url "https://github.com/VOICEVOX/voicevox/releases/download/#{version}/VOICEVOX.#{version}.dmg",
+      verified: "github.com/VOICEVOX/voicevox/"
+  name "VOICEVOX"
+  desc "Free, medium-quality text-to-speech software"
+  homepage "https://voicevox.hiroshiba.jp/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
+  conflicts_with cask: "voicevox-preview"
+
+  app "VOICEVOX.app"
+
+  zap trash: [
+    "~/Library/Application Support/voicevox",
+    "~/Library/Application Support/voicevox-cpu",
+    "~/Library/Application Support/voicevox-engine",
+    "~/Library/Logs/voicevox-cpu",
```

Comment:
次辺りにディレクトリが増える（"変わる"じゃなく"増える"なのは[不具合](https://github.com/VOICEVOX/voicevox/issues/1487)ですが。。）ので追加しちゃいましょう！

```suggestion
    "~/Library/Logs/voicevox",
    "~/Library/Logs/voicevox-cpu",
```

Reply:
次のリリースのときに追加するPRを出そうと思います！ (ここをActionsで自動更新させるのは難しそうですね…)